test/tests/coord_type/tests

```diff
@@ -5,7 +5,7 @@
     input = coord_type_rz.i
     exodiff = coord_type_rz_out.e
   [../]
-  [/rz-x-rotation]
+  [./rz-x-rotation]
```

Comment:
Looks like we need some error checking in the TestHarness. This would have triggered an error in MOOSE.